### ISSUE TITLE
Xcode 7.2: cannot build app from the command line

### ISSUE THREAD

**Issue by jmoody** [calabash/ios-smoke-test-app#52]
```
$ make clean
$ make app-cal
▸ Building ios-smoke-test-app/CalSmoke-cal [Debug]
▸ Check Dependencies

:x:  CodeSign error: entitlements are required for product type 'Application' in SDK 'Simulator - iOS 9.2'. Your Xcode installation may be damaged.
```

**Comment by mikeposh** [comment]
We have also been hit by this when running our Calabash tests and in fact have struggled to get rid of the error even by re-installing Xcode 7.1.1 - I was wondering if some of the other OS X updates which arrived at the same time as Xcode 7.2 have something to do with it.

Have you any inkling at all as to what might cause this?

**Comment by richlong** [comment]
Getting same error here running Xcode 7.2. Both targets build and run fine within xcode just not using "make app-cal" through terminal, using the same profile for signing. "make ipa" commands runs successfully.

**Comment by jmoody** [comment]
Also related:

* Xcode 7.2: builds simulator dylib to Debug-iphoneos directory [#303](https://github.com/calabash/calabash-ios-server/issues/303)
* Xcode 7.2: builds frank simulator library to Debug-iphoneos directory [#302](https://github.com/calabash/calabash-ios-server/issues/302)

**Comment by jmoody** [comment]
ATTN @krukow

**Comment by jmoody** [comment]
@richlong @mikeposh 

For this project, Xcode builds stage binaries to the same place that the command line scripts stage the binaries - `./Products/`.  Until we can solve this Xcode 7.2 problem, see if you can integrate this behavior into your projects.

**Comment by jmoody** [comment]
Update:  I tried a few things today and nothing worked.

In desperation, I cashed in one of my Apple Support tickets for code-level support.  Waiting on a response.

**Comment by jmoody** [comment]
Apple support has been disappointing.

**Comment by jmoody** [comment]
I am updating the build scripts in this repo:

https://github.com/jmoody/briar-ios-example

and found that I could build with Xcode 7.2!

However, after I made some changes to the project:

* Add missing icons
* Add missing Launch.xib

I could no longer build because of this error.

**Comment by jmoody** [comment]
I rolled back, now I cannot reproduce the successful builds.

**Comment by jmoody** [comment]
@mikeposh @richlong 

I think I have a solution based on feedback from my Apple Direct Support Ticket.

https://github.com/calabash/Permissions/pull/12

Can either of you take a moment and follow the `To Review` instructions in that PR on the Permissions project?

If that PR works, I have a likely solution for this repo.

Thanks!

**Comment by jmoody** [comment]
Fixed with #54 !

**Comment by lummyare** [comment]
Getting make: *** [app-cal] Error 65

**Comment by mikeposh** [comment]
@jmoody - thanks for using your support ticket and getting a solution for this. Nice to have this working again.

**Comment by chaoscope** [comment]
@jmoody I am getting this same issue trying to build with a workspace + scheme, is the only solution to use a project + target?
